packages/core/package.json

```diff
@@ -21,6 +22,9 @@
     "access": "public"
   },
   "scripts": {
+    "build": "tsc",
+    "clean": "tsc --build --clean",
+    "dev": "tsc-watch --noClear",
```

Comment:
should we utilise `tsc-watch` as intended and string up some command development commands in the root `package.json` for use?

e.g.
- `tsc-watch .... --onSuccess node ......./cli/index.js ./docs` (docs development)
- maybe also one for the default template site

to reduce the slightly more cumbersome 2-step process to one

otherwise, also somewhat equivalent to just using [`tsc --watch`](https://www.typescriptlang.org/docs/handbook/compiler-options.html). (`tsc-watch` is meant as a typescript substitute for [nodemon](https://www.npmjs.com/package/nodemon))

Reply:
I have thought of this, but I am not sure on how taxing would it be when MarkBind commands are repeatedly invoked in short bursts over the course of development. For example, `serve` needs time to build, and it's not ideal for the script to call it in every added line when I am in the middle of implementing a function that takes some time to develop, it might just hog resources with only a little convenience benefit.

However, I am admittedly not too experienced in how `tsc-watch` does things (or `nodemon` too), so I am also not sure whether this is a real issue 🤔 but I have tried this with just `markbind --version` and it is invoked multiple times as I write some lines at a relaxed pace (the cutoff time delay is ~1 second).

Reply:
Good point, I think we can keep the 2 commands separate for now and see how things go. (also just realised `--onSuccess` and the like won't work without having resolved all the typescript errors)

In that sense, would `tsc --watch` be sufficient for now? (over `tsc-watch`)

Reply:
Yeah, it would probably be `--onCompilationComplete` if we want to achieve this in the middle of migration.

I use the package `tsc-watch` just for the `--noClear` option for now, though it can be discussed whether it would be necessary for us. Just my 2 cents, in my command prompt, I don't quite like how `tsc --watch` clears the screen when it starts, so the previous command results and all are gone. I think it's better for the history to still be viewable by devs. What do you think?

Reply:
Would all compilation errors be reported again by `tsc --watch`? (or just incremental ones) I personally don't have much preference if so, since we should only need one `tsc` process for all packages. 

If not, I think `tsc-watch` is definitely the way to go 👍 (fine with either though, if it might be useful to some developers let's leave it as is)

Reply:
On further consideration, I think `tsc-watch` is not needed for now, and can opt for `tsc --watch` first. Turns out the clearing the screen thing is more on the terminal and not on the `tsc`'s shortcomings 😃